coreos-installer

```diff
@@ -724,6 +724,7 @@ write_image_to_dasd() {
     # low-level format the ECKD DASD using dasdfmt, if needed
     DEST_DEV_BUS=$(lszdev --by-node ${DEST_DEV} | tail -n 1  | awk '{print $2}')
     if [ "$(< /sys/bus/ccw/devices/${DEST_DEV_BUS}/status)" = "unformatted" ]; then
+        log "Formatting DASD device ${DEST_DEV_BUS}"
```

Comment:
minor nit: seems topic-wise a bit unrelated, but I'm OK with this additional changed line in the DASD special handling

Reply:
Yep :D hijacking this PR a bit for this line.